elements/ethernet/ensureether.hh

```diff
@@ -24,19 +24,15 @@ specified by the arguments is prepended to the packet.
 
 =e
 
-Encapsulate packets in an Ethernet header with type
+Encapsulate packets without an Ethernet header with type
 ETHERTYPE_IP (0x0800), source address 1:1:1:1:1:1, and
 destination address 2:2:2:2:2:2:
 
-  EtherEncap(0x0800, 1:1:1:1:1:1, 2:2:2:2:2:2)
-
-=n
-
-For IP packets you probably want to use ARPQuerier instead.
+  EnsureEther(0x0800, 1:1:1:1:1:1, 2:2:2:2:2:2)
 
 =a
 
-EtherEncap */
+EnsureEther */
```

Comment:
Hm, this doc field is usually used for “see also”, it's weird for an element to refer to itself.